app/routes/screens/race/Race.js

```diff
@@ -101,21 +103,31 @@ class RacePredictor extends Component {
             />
           )}
           {race.locations &&
-            Object.entries(race.locations).map(([id, location]) =>
-              <MapView.Marker
-                coordinate={location.coordinates}
-                title={id}
-                description={new Date(location.time).toLocaleString()}
-                pinColor={`#${RacePredictor.intToColor(id)}`}
-                key={id}
-              />
-            )}
+            Object.entries(race.locations).map(([id, location]) => {
+              const coordinate = {
+                longitude: location.coords.longitude,
+                latitude: location.coords.latitude
+              };
+
+              return (
+                <MapView.Marker
+                  coordinate={coordinate}
+                  title={id}
+                  description={new Date(location.timestamp).toLocaleString()}
+                  pinColor={`#${RacePredictor.intToColor(id)}`}
```

Comment:
intToColor -> not static should be an helper its meant to be shared between components